File: src/config/production.ts
import { Config } from "./Config";

const config: Config = {
  resqApiBaseUrl: "https://stroke.qualityregistry.org/api/rest/hpfdc/",
  keycloakAuthUrl: "https://auth.qualityregistry.org/realms/stroke/protocol/openid-connect/auth",
  oauthClientId: "doc-marker",
  uploadServerBaseUrl: "https://quest.ms.mff.cuni.cz/resq-doc-marker/backend/",
  uploadServerActInDevelopment: false,
  resqRecordUrl: (id) => `https://stroke.qualityregistry.org/data-collection/record/${id}/form`,
};

export default config;

File: backend/app/config/production.py
from .Config import Config
from pathlib import Path

production_config = Config(
    resq_api_base_url="https://stroke.qualityregistry.org/api/rest/hpfdc/",
    uploaded_files_storage_path=Path("storage", "production"),
    keycloak_token_url="https://auth.qualityregistry.org/realms/stroke/protocol/openid-connect/token",
    oauth_client_id="doc-marker",
    oauth_client_password=None,
)
